### ISSUE TITLE
bug: Response for insufficient quota

### ISSUE THREAD

**Issue by sharansukesh1003** [sharansukesh1003/GPT-Mobile#1]
Implement error handling for insufficient search quota

**Comment by mohitdebian** [comment]
probably it's the issue with your open ai account. It's showing this error because your open ai account usage limit is now full and now on you can't use your open ai account and if you want to use it either pay for it or make anew free account with another email id

**Comment by sharansukesh1003** [comment]
> probably it's the issue with your open ai account. It's showing this error because your open ai account usage limit is now full and now on you can't use your open ai account and if you want to use it either pay for it or make anew free account with another email id

Hey, thanks for addressing the issue..btw I figured that issue and just I want to handle the error in other ways.. like showing the user that their account is no longer eligible for free queries. Currently it shows invalid API key for the aforementioned error.

**Comment by mohitdebian** [comment]
> > probably it's the issue with your open ai account. It's showing this error because your open ai account usage limit is now full and now on you can't use your open ai account and if you want to use it either pay for it or make anew free account with another email id
> 
> Hey, thanks for addressing the issue..btw I figured that issue and just I want to handle the error in other ways.. like showing the user that their account is no longer eligible for free queries. Currently it shows invalid API key for the aforementioned error.

Yes i was saying the same thing but in a different way 😊. I am glad that your problem is now solved
